patches/minecraft/net/minecraft/entity/EntityLivingBase.java.patch

```diff
@@ -231,15 +231,17 @@
          super.func_70071_h_();
          this.func_184608_ct();
  
-@@ -2015,6 +2046,7 @@
+@@ -2014,7 +2045,9 @@
+ 
                  if (!ItemStack.func_77989_b(itemstack1, itemstack))
                  {
++                    if (!ItemStack.areItemStacksEqualUsingNBTShareTag(itemstack1, itemstack))
```

Comment:
I would argue that the event should still be fired as that is fired server side and thus people can possibly hook into that if they want to react to Equipment changes which may involve nbt or capability changes that are not synced to client.
Or is this event supposed to only ever fire when the SPacketEntityEquipment packet is sent?